Automattic/pocket-casts-ios
# [928] Visually mark played episodes in search results

Fixes #928

This PR replicates the UI experience for episodes marked as played in the Search Results Episodes section:

![search-results-episodes](https://github.com/Automattic/pocket-casts-ios/assets/912252/39835a32-0fcc-41de-903c-809573b9c2a9)

## To test

• CI should be 🟢 
• Run the app and go to _Discover_
• Search for some Podcasts, like _Radio Swatt_ or any other Podcast
• If the search result contains episodes marked as played, you should see the new style applied
• If not you can tap on an episode to manually mark it as played
• Reload the search and wait for the result
• You should see the same result as before but with the episode marked as played

## Checklist
